### DISCUSSION TITLE
绘制和量算工具提示版本不正常，以及绘制不显示

### DISCUSSION THREAD

**Discussion by MickeyMouse623** [zouyaoji/vue-cesium#509] [Q&A]
1、提示Cesium版本问题，但已经是最新版本
<img width="920" alt="image" src="https://github.com/zouyaoji/vue-cesium/assets/18524460/fba7ed22-3a20-4212-9486-5db9c3771277">

2、绘制和量算在1300000米以上绘制时，绘制的图标点、线段、多边形都不显示
步骤一、先在默认视角下绘制两个图标点，但是不显示
步骤二、地图缩放到130W米以下再绘制一个图标时，3个图标点都显示
步骤三、移动地图时，步骤一绘制的两个点跟随地图移动，步骤二的不移动
<img width="494" alt="image" src="https://github.com/zouyaoji/vue-cesium/assets/18524460/db7fb5d3-7758-4246-8a83-091cf2a5d0cc">
<img width="664" alt="image" src="https://github.com/zouyaoji/vue-cesium/assets/18524460/952d0e16-562a-41cc-802a-1a73cc3d8008">

**Comment by zouyaoji** [comment]
下个版本修复。
